src/coredump/coredump.c

```diff
@@ -286,16 +286,21 @@ static int make_filename(const char *context[_CONTEXT_MAX], char **ret) {
         if (!p)
                 return -ENOMEM;
 
+        s = filename_escape(context[CONTEXT_SIGNAL]);
+        if (!s)
+                return -ENOMEM;
```

Comment:
I don't think this change is useful. Those names are very long already, and making them even longer is not nice.
The signal that killed the process is already recorded in the core file itself, so even if the extended attribute is lost, this information is available.